src/main/java/org/mastodon/revised/trackscheme/display/DefaultTrackSchemeOverlay.java

```diff
@@ -448,9 +704,98 @@ protected void drawVertexFull( final Graphics2D g2, final ScreenVertex vertex )
 			spotdiameter *= ( 1 + ratio );
 		final double spotradius = spotdiameter / 2;
 
+		// Unwrap to read number of incoming.outgoing edges.
+		final TrackSchemeVertex ref = graph.vertexRef();
+		final TrackSchemeVertex tsv = graph.getVertexPool().getObject( vertex.getTrackSchemeVertexId(), ref );
+		final int[] incomingIDs = new int[ tsv.incomingEdges().size() ];
+		int i1 = 0;
+		for ( final TrackSchemeEdge tse : tsv.incomingEdges() )
+			incomingIDs[ i1++ ] = tse.getModelEdgeId();
+		final int[] outgoingIDs = new int[ tsv.outgoingEdges().size() ];
+		int i2 = 0;
+		/*
+		 * FIXME: Often, the JVM crashes on this line (or equivalent in the
+		 * other vertex methods) with the following error:
```

Comment:
It only occurs on windows, probably because the rendering is faster on this platform. I will investigate more.